pandas/io/tests/parser/test_network.py

```diff
@@ -27,51 +34,15 @@ def setUp(self):
                          'pandas/io/tests/parser/data/salaries.csv')
 
     @tm.network
-    def test_url_gz(self):
-        url = self.base_url + '.gz'
-        url_table = read_table(url, compression="gzip", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
-
-    @tm.network
-    def test_url_bz2(self):
-        url = self.base_url + '.bz2'
-        url_table = read_table(url, compression="bz2", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
-
-    @tm.network
-    def test_url_xz(self):
-        url = self.base_url + '.xz'
-        url_table = read_table(url, compression="zip", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
-
-    @tm.network
-    def test_url_zip(self):
-        url = self.base_url + '.zip'
-        url_table = read_table(url, compression="zip", engine="python")
-        tm.assert_frame_equal(url_table, self.local_table)
+    def test_compressed_urls(self):
```

Comment:
Cool, I see the test generator examples in [`io/tests/test_pickle.py`](https://github.com/pandas-dev/pandas/blob/06b35db1d88e75a9e3f183b58678589cf7715381/pandas/io/tests/test_pickle.py#L194), [`computation/tests/test_compat.py`](https://github.com/pandas-dev/pandas/blob/06b35db1d88e75a9e3f183b58678589cf7715381/pandas/computation/tests/test_compat.py#L40), and [`computation/tests/test_eval.py`](https://github.com/pandas-dev/pandas/blob/06b35db1d88e75a9e3f183b58678589cf7715381/pandas/computation/tests/test_eval.py#L809). Those examples don't have a `setUp` method nor do they use the `pandas.util.testing.network` decorator. So I did my best in 0fcb3c328fcec8b1beb1ed8c968ed4c7835e5da1. It seems that one side effect is that the failure messages are less informative?